cmd/mattermost/commands/plugin.go

```diff
@@ -184,3 +215,83 @@ func pluginListCmdF(command *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func pluginPublicKeysCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	verbose, err := command.Flags().GetString("verbose")
+	if err != nil {
+		return errors.New("failed reading verbose. Error: " + err.Error())
```

Comment:
Should we use `errors.Wrap` instead of concatenating the string?
```suggestion
		return errors.Wrap(err, "failed reading verbose flag")
```